model/project.go

```diff
@@ -493,6 +497,7 @@ type ProjectTask struct {
 	DependsOn       []TaskUnitDependency `yaml:"depends_on,omitempty" bson:"depends_on"`
 	Commands        []PluginCommandConf  `yaml:"commands,omitempty" bson:"commands"`
 	Tags            []string             `yaml:"tags,omitempty" bson:"tags"`
+	Distros         []string             `yaml:"distros,omitempty" bson:"distros"`
```

Comment:
I think this is useful to add, but just wondering what your motivation for adding it in this pr is?

Reply:
I mentioned this in the top-level comment: I remembered seeing a ticket for it and it was easier to add now since I was already tracing that code path.  (https://jira.mongodb.org/browse/EVG-14269)